go-sdk/common/transform.go

```diff
@@ -0,0 +1,48 @@
+// Copyright 2024 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package common
+
+type TransformKind = string
+
+const (
+	JoinByColumValueKind    TransformKind = "JoinByColumnValue"
```

Comment:
does it really makes sense this is in the package common ? In cue SDK it's not in the common package but directly into the table plugin.

I think we should be coherent and have the same thing in both case. From my point of view, this should be part of the table plugin

Reply:
Will move it to common, so that can be used by other panels 😄